static/js/publisher/pages/Releases/helpers.ts

```diff
@@ -47,6 +47,21 @@ export function getRevisionsArchitectures(revisions: any[]) {
   return archs;
 }
 
+export function getLatestChannelRevisionRelease(
```

Comment:
For me the name suggest the function would return "latest channel", why not naming it `getLatestRelease` (the attributes should be enough to imply the filtering) or `getLatestReleaseForRevisionInChannel` (if we want to be explicit about attributes).